packages/editor/src/lib/exports/getSvgJsx.tsx

```diff
@@ -0,0 +1,383 @@
+import {
+	TLFrameShape,
+	TLGroupShape,
+	TLShape,
+	TLShapeId,
+	getDefaultColorTheme,
+} from '@tldraw/tlschema'
+import { hasOwnProperty, promiseWithResolve } from '@tldraw/utils'
+import {
+	ComponentType,
+	Fragment,
+	ReactElement,
+	useEffect,
+	useLayoutEffect,
+	useMemo,
+	useRef,
+	useState,
+} from 'react'
+import { flushSync } from 'react-dom'
+import { ErrorBoundary } from '../components/ErrorBoundary'
+import { InnerShape, InnerShapeBackground } from '../components/Shape'
+import { Editor, TLRenderingShape } from '../editor/Editor'
+import { ShapeUtil } from '../editor/shapes/ShapeUtil'
+import {
+	SvgExportContext,
+	SvgExportContextProvider,
+	SvgExportDef,
+} from '../editor/types/SvgExportContext'
+import { TLImageExportOptions } from '../editor/types/misc-types'
+import { useEditor } from '../hooks/useEditor'
+import { useEvent } from '../hooks/useEvent'
+import { Box } from '../primitives/Box'
+import { Mat } from '../primitives/Mat'
+import { ExportDelay } from './ExportDelay'
+
+export function getSvgJsx(editor: Editor, ids: TLShapeId[], opts: TLImageExportOptions = {}) {
+	if (!window.document) throw Error('No document')
+
+	const {
+		scale = 1,
+		background = false,
```

Comment:
i know this code precedes you but a comment explaining what `background` does would be helpful.